CHANGELOG.md

```diff
@@ -1,25 +1,56 @@
-### upcoming 2.28.0 (April 6, 2023)
+### upcoming 2.29.0 (----)
 
 **New Features:**
 
-- added Show Output Channel command [#568](https://github.com/vshaxe/vshaxe/issues/568)
+- added Fix All command ([#576](https://github.com/vshaxe/vshaxe/pull/576)). You can enable autofixes on save for simple compiler errors, like missing semicolons, see [Fix All](https://github.com/vshaxe/vshaxe/wiki/Commands#haxe-fix-all)
+- added `-D message-reporting=pretty` mode support to build tasks ([#587](https://github.com/vshaxe/vshaxe/pull/587))
+- added quick fix for `modifier is not supported for module-level fields` ([haxe-languageserver#92](https://github.com/vshaxe/haxe-language-server/pull/92))
+- added `Extract to var in enclosing scope` action ([haxe-languageserver#95](https://github.com/vshaxe/haxe-language-server/pull/95)), see [example](https://github.com/vshaxe/vshaxe/wiki/Code-Actions#extract-var-to-enclosing-scope)
+- added `Change final to var` quick fix ([haxe-languageserver#96](https://github.com/vshaxe/haxe-language-server/pull/96))
+- added update syntax code actions for null operators: ([haxe-languageserver#97](https://github.com/vshaxe/haxe-language-server/pull/97))
```

Comment:
Seems very verbose for a changelog entry? We usually have one liners or not much longer than that